fairmq/plugins/control/Control.cxx

```diff
@@ -65,13 +65,18 @@ Control::Control(const string& name, Plugin::Version version, const string& main
     });
 
     try {
-        TakeDeviceControl();
-
         auto control = GetProperty<string>("control");
 
+        if (control != "none") {
+            TakeDeviceControl();
+        }
+
         if (control == "static") {
             LOG(debug) << "Running builtin controller: static";
             fControllerThread = thread(&Control::StaticMode, this);
+        } else if (control == "none") {
+            LOG(debug) << "Running builtin controller: none";
+            fControllerThread = thread(&Control::NoControlMode, this);
```

Comment:
Hm, let's not start a control thread at all?

Reply:
Done!